.azurepipelines/azure-pipelines.yml

```diff
@@ -89,15 +89,24 @@ jobs:
         Build.Name:   "adls"
         Build.Arch:   ""
         Build.Target: ""
-      IDV_X64_DEBUG:
-        Build.Name:   "idv"
+      ADLP_X64_RELEASE:
+        Build.Name:   "adlp"
+        Build.Arch:   "-a x64"
+        Build.Target: "-r"
+      ADLPS_X64_DEBUG:
+        Build.Name:   "adlps"
```

Comment:
Both ADL-P and ADL-PS would build using X64. maybe change one of them to X86 since these 2 platforms are close?

Reply:
Changed